client/src/DecProvider.ts

```diff
@@ -2,42 +2,38 @@ import { pathToFileURL } from "url";
 import { CancellationToken, Declaration, DeclarationProvider, Location, Position, ProviderResult, TextDocument, Uri } from "vscode";
 import { RequestType, VersionedTextDocumentIdentifier } from "vscode-languageclient";
 import Client from "./client";
-import { CompletionItemCoq, CompletionItemCoqRequest, CompletionItemCoqResponse } from "./protocol/types";
+import { CompletionItemCoq, CompletionItemCoqRequest, CompletionItemCoqResponse, DeclarationLocationCoqRequest, DeclarationLocationCoqResponse } from "./protocol/types";
+import * as fs from 'fs';
```

Comment:
I think we prefer explicit imports, though I do get that associating it with the namespace improves legibility somewhat.
```suggestion
import { existsSync } from 'fs';
```